### DISCUSSION TITLE
Lesson 3:  compilerMetadata is modifyingcontracts/artifacts/FundMe.json

### DISCUSSION THREAD

**Discussion by jafaritaqi** [smartcontractkit/full-blockchain-solidity-course-py#161] [Q&A] [answered]
at 2:46:16 when I hit compile FundMe.sol I get this error :  compilerMetadata is modifyingcontracts/artifacts/FundMe.json anyone can help
![Screenshot from 2021-10-09 15-58-12](https://user-images.githubusercontent.com/9637377/136658828-ade8227a-caea-4f87-888a-d2b51c342cfc.png)
?

**Comment by PatrickAlphaC** [comment, marked as answer]
Does anything here help? https://ethereum.stackexchange.com/questions/106449/compilermetadata-is-modifying-artifacts-name-of-contract-json
